benchmark/large/other/or_filter_pushdown.benchmark

```diff
@@ -0,0 +1,15 @@
+# name: benchmark/large/other/or_filter_pushdown.benchmark
```

Comment:
Happy to remove this benchmark if necessary. Was meant mostly for measuring the improvements